From: Jun Yang <jun.yang@suse.com>
Date: Thu, 10 Feb 2022 18:02:29 +0800
Subject: [PATCH] netswift: fix compile issue of txgbe_main.c
Patch-mainline: Never, fixes out-of-tree driver
References: bsn#92
Modified-by-SEL: N/A

commit 30bfce109420912f201d4f295f9130ff44f04b41
(net: remove ndo_udp_tunnel_* callbacks)
remvoe ndo_upd_tunnel_add and nod_upd_tunnel_del.

This patch remove ndo_udp_tunnel_* in netswift driver
to fix compile issue follow above commit.

Signed-off-by: Jun Yang <jun.yang@suse.com>
---
 drivers/net/ethernet/netswift/txgbe/txgbe_main.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/net/ethernet/netswift/txgbe/txgbe_main.c b/drivers/net/ethernet/netswift/txgbe/txgbe_main.c
index 5e21f1a22984..5d371c5bec7c 100644
--- a/drivers/net/ethernet/netswift/txgbe/txgbe_main.c
+++ b/drivers/net/ethernet/netswift/txgbe/txgbe_main.c
@@ -7427,8 +7427,6 @@ static const struct net_device_ops txgbe_netdev_ops = {
 	.ndo_fdb_add            = txgbe_ndo_fdb_add,
 	.ndo_bridge_setlink     = txgbe_ndo_bridge_setlink,
 	.ndo_bridge_getlink     = txgbe_ndo_bridge_getlink,
-	.ndo_udp_tunnel_add     = txgbe_add_udp_tunnel_port,
-	.ndo_udp_tunnel_del     = txgbe_del_udp_tunnel_port,
 	.ndo_features_check     = txgbe_features_check,
 	.ndo_set_features       = txgbe_set_features,
 	.ndo_fix_features       = txgbe_fix_features,
-- 
2.34.1

